main/templates/base.html

```diff
@@ -108,6 +108,23 @@
 			<div class="text-center" id="copyrightMessage">Copyright &copy; Enactus Waterloo 2015.</div>
 		</div>
 	</footer>
+<script>
```

Comment:
This entire script uses indentation differently than the rest of the project, so the script indentation needs to be fixed.